Library/Tracking/KSRAnalytics.swift

```diff
@@ -413,6 +413,19 @@ public final class KSRAnalytics {
     }
   }
 
+  /**
+   An optional context that provides an addtional context  about the location of the event.
```

Comment:
Just to clarify, the context itself is not optional but when you use it as a parameter it is. Replace with:
`A context providing additional details about the location the event occurs.`